src/games/strategy/triplea/util/TransportUtils.java

```diff
@@ -198,4 +213,18 @@ public int compare(final Unit o1, final Unit o2) {
     return Optional.empty();
   }
 
+  private static void loadMaxUnits(final Unit transport, final List<Unit> canBeTransported,
```

Comment:
The calling method is a bit better, but we've also taken a step back as well. This method is simply and effectively not in-lined anymore. That does not help so much since you need to go down and still read this method, it does not succeed in any information hiding. The small step back, we have some code duplication in the method.

The reason for writing tests is to force a clean design, I recommend taking that approach, so there is some real pressure to have a clean, testable design. 

Failing any updates, if I were to start refactoring this, an eventual step I would probably take would be to inline this method, eliminate the duplicated data.